iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -112,6 +112,14 @@ LogicalResult verifyDoubleTilingExpertPassPipelineConfig(
   IREE::Flow::PartitionableLoopsInterface interfaceOp =
       dyn_cast_or_null<IREE::Flow::PartitionableLoopsInterface>(op);
   if (interfaceOp) {
+    SmallVector<int64_t> firstLevelTileSizes = loweringConfig.getTileSizeVals(
```

Comment:
Could we relax this a bit. If unset the inner loops are treated as tile size set to 0. I dont think we need to explicitly enforce this?